src/wallet/wallet2.h

```diff
@@ -1062,7 +1062,7 @@ namespace tools
     }
 
     void set_ring_database(const std::string &filename);
-    const std::string get_ring_database() const { return m_ring_database; }
+    std::string get_ring_database();
```

Comment:
Why is it not const ?

Reply:
Not sure tbh, think the build was failing when it was in there.